bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/handler/PulseaudioHandler.java

```diff
@@ -218,67 +213,25 @@ public void dispose() {
 
     @Override
     public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
-        if (bridgeStatusInfo.getStatus() == ThingStatus.ONLINE
-                && getThing().getStatusInfo().getStatusDetail() == ThingStatusDetail.BRIDGE_OFFLINE) {
-            // Bridge is now ONLINE, restart the refresh job to get an update of the thing status without waiting
-            // its next planned run
-            ScheduledFuture<?> job = refreshJob;
-            if (job != null && !job.isCancelled()) {
-                job.cancel(true);
-                refreshJob = null;
-            }
-            deviceOnlineWatchdog();
-        } else if (bridgeStatusInfo.getStatus() == ThingStatus.OFFLINE
+        if (bridgeStatusInfo.getStatus() == ThingStatus.OFFLINE
                 || bridgeStatusInfo.getStatus() == ThingStatus.UNKNOWN) {
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+        } else if (bridgeStatusInfo.getStatus() == ThingStatus.UNINITIALIZED) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED);
```

Comment:
I am not sure this case can happen, I mean `bridgeStatusChanged` called with `ThingStatus.UNINITIALIZED`

Reply:
You are right. I tried and it's not. Removed.

Reply:
Now, compared to the method you override, the difference is that you are not setting the status to ONLINE when bridge is ONLINE. Is it what you want ? That would mean that the update to ONLINE is managed somewhere else.

Reply:
Yes. Because if the bridge is ONLINE, it doesn't mean that its child are.
The pulseaudio child devices are set to ONLINE by the deviceUpdate() method, which is called by the scheduled update() method in the PulseaudioBridgeHandler.

So, when the pulseaudio bridge is back online (it could only be done by the scheduled update method() in the PulseaudioBridgeHandler class), it calls (in the same update method) the pulseaudio server to request info and when it has info, it immediately updates its child accordingly (either ONLINE or OFFLINE).